masterlist.yaml

```diff
@@ -946,6 +946,78 @@ plugins:
     url: ['https://www.nexusmods.com/skyrimspecialedition/mods/2259/']
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
+  - name: "JKs Skyrim.esp"
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/6289/']
+    inc:
+      - 'Blues Skyrim.esp' # DAWN OF SKYRIM, patch is planned
+      - 'Open Cities Skyrim.esp'
+      - 'Paradise_City_Merged.esp'
+      - 'Paradise_City_MergedNoWindhelm.esp'
+      - 'Paradise_City_MergedSummerWindhelm.esp'
+      - 'Paradise_City_Rorikstead.esp'
```

Comment:
Can conditionals be added on inc? 
Have two versions of the same mod, same esp name. One is compatible.
A warning message & checksum comparison will be fine anyway.

Reply:
Yes, inc entries are [Files](https://loot-api.readthedocs.io/en/latest/metadata/data_structures/file.html).

Reply:
Thanks, conditional works fine, display doesn't seem to work though. I'll add an additional warn message instead with link to compatible version.

    inc:
      - name: 'Friendlier Taverns.esp'
        condition: 'active("Friendlier Taverns.esp") and checksum("Friendlier Taverns.esp",0C38A36A)'
        display: '[Friendlier Taverns with Baths and Barns SSE](https://www.nexusmods.com/skyrimspecialedition/mods/1399/)'

Reply:
Display should, if it doesn't it's a bug, so don't work around it.

Reply:
Well I have a additional message  which points to compatible version. I will add in the display again now.